From rdunlap@xenotime.net Fri Oct 28 20:45:02 2005
Date: Fri, 28 Oct 2005 20:36:51 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
To: gregkh <greg@kroah.com>
Subject: pci-driver: store_new_id() not inline
Message-Id: <20051028203651.10d46653.rdunlap@xenotime.net>

From: Randy Dunlap <rdunlap@xenotime.net>

store_new_id() should not be (and cannot be) inline;
the function pointer is stored in a device_attribute table.

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
 drivers/pci/pci-driver.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/pci/pci-driver.c	2005-11-02 09:25:02.000000000 -0800
+++ gregkh-2.6/drivers/pci/pci-driver.c	2005-11-02 11:54:14.000000000 -0800
@@ -36,7 +36,7 @@
  * Adds a new dynamic pci device ID to this driver,
  * and causes the driver to probe for all devices again.
  */
-static inline ssize_t
+static ssize_t
 store_new_id(struct device_driver *driver, const char *buf, size_t count)
 {
 	struct pci_dynid *dynid;
